cleanup-workflows.sh

```
@@ -0,0 +1,47 @@
+PRESERVE_FILES=(
+  "core-ci.yml"
+  "core-deploy.yml"
+  "core-security.yml"
+  "README.md"
+)
+
+echo "Archiving unnecessary GitHub Actions workflows..."
+echo ""
+
+# Move each workflow file to archive directory if not in preserve list
+cd "$WORKFLOWS_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)